core/node/groups.go

```diff
@@ -122,6 +122,12 @@ func LibP2P(bcfg *BuildCfg, cfg *config.Config) fx.Option {
 			logger.Error("Use the `Swarm.Transports.Network.Relay' config field instead")
 		}
 	}
+	//nolint
+	if cfg.Swarm.EnableRelayHop {
+		logger.Error("The `Swarm.EnableRelayHop` config field is deprecated.\n" +
+			"Use `Swarm.RelayService` to configure the circuit v2 relay.\n" +
+			"If you want to continue running a circuit v1 relay, please use the standalone relay daemon: https://github.com/libp2p/go-libp2p-relay-daemon\n")
+	}
```

Comment:
I believe we should simply refuse to start to avoid people wasting time wondering why relay stopped working:
```suggestion
        os.Exit(1)
	}
```

Reply:
i agree.